Update/onik_010.txt

```diff
@@ -913,7 +913,7 @@ void main()
 	OutputLine(NULL, "　む、あるとの答えが２人しか出なかったぞ？",
 		   NULL, "Hrm, only two people said they had one?", Line_WaitForInput);
 	OutputLine(NULL, "　ってことは…「レナ」「斧」「ラウンジ」のどれかひとつは…犯人、正解ということか？！",
-		   NULL, " So it means... out of \"Rena\", \"Axe\", and \"Lounge,\" one of them is correct and is part of the crime!?", Line_Normal);
+		   NULL, " So it means... out of \"Rena,\" \"Axe,\" and \"Lounge,\" one of them is correct and is part of the crime!?", Line_Normal);
```

Comment:
I'm no expert on quoting but are you sure about this? These two commas are a bit different than the last one semantically.

Reply:
seems like commas and dots are all the time included.